test/synchronicity_test.py

```diff
@@ -490,3 +491,47 @@ async def get_self(self):
     self_from_aio_interface = await original.get_self.aio()
     assert self_from_aio_interface == original
     assert isinstance(self_from_aio_interface, BlockingFoo)
+
+
+def test_no_input_translation(monkeypatch):
+    s = Synchronizer()
+
+    @s.create_blocking
+    def does_input_translation(arg: float) -> str:
+        return str(arg)
+
+    @s.create_blocking
+    @s.no_input_translation
+    async def without_input_translation(arg: float) -> str:
+        return str(arg)
+
+    in_translate_spy = MagicMock(wraps=s._translate_scalar_in)
+    monkeypatch.setattr(s, "_translate_scalar_in", in_translate_spy)
+    does_input_translation(3.14)  # test without decorator, this *should* do input translation
+    in_translate_spy.assert_called_once_with(3.14)
+
+    in_translate_spy.reset_mock()
+    without_input_translation(3.14)  # test without decorator, this *should* do input translation
+    in_translate_spy.assert_not_called()
+
+
+def test_no_output_translation(monkeypatch):
+    s = Synchronizer()
+
+    @s.create_blocking
+    def does_input_translation(arg: float) -> str:
```

Comment:
Should this be `does_output_translation`?

Reply:
Yup :)